cves/kernel/CVE-2014-3183.yml

```diff
@@ -305,9 +357,19 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: false
+  note: |
+    According to https://bugs.chromium.org/p/project-zero/issues/detail?id=90,
```

Comment:
Nice job on this. Clearly explains the any significant talk about the vulnerability